addons/web/static/lib/pdfjs/src/core/core.js

```diff
@@ -93,7 +93,7 @@ var Page = (function PageClosure() {
       // "The crop, bleed, trim, and art boxes should not ordinarily
       // extend beyond the boundaries of the media box. If they do, they are
       // effectively reduced to their intersection with the media box."
-      cropBox = Util.intersect(cropBox, mediaBox);
+      cropBox = PDFJS.Util.intersect(cropBox, mediaBox);
```

Comment:
@jeh-odoo @pko-odoo 

I don't know which one of you made this change since you both worked on this PR, so I ping you both.

Please be careful with this, you're not supposed to change code in third party libraries (pdfjs in this case).
These libraries could be updated at any time and your code will simply be gone (and the bug re-introduced).

If you're interested in the actual fix for this problem, you can check: https://github.com/odoo/odoo/pull/35518